docs/book/src/images/worker-machines-resources.plantuml

```diff
@@ -0,0 +1,48 @@
+@startuml worker-machines-resources
+
+skinparam nodesep 80
+skinparam ranksep 80
+skinparam object {
+    backgroundColor<< Infrastructure provider CRD >> #D9EAD3
+    backgroundColor<< Bootstrap provider CRD >> #FEF4F4
+}
+
+' Cluster API CRD
+object Cluster << Cluster API CRD >>
+object MachineDeployment << Cluster API CRD >> {
+    spec.clusterName
+    <font color="Green">spec.template.spec.infrastructureRef</font>
+    <font color="Red">spec.template.spec.bootstrap.configRef</font>
+}
+object MachineSet << Cluster API CRD >> {
+    <font color="Green">spec.template.spec.infrastructureRef</font>
+    <font color="Red">spec.template.spec.bootstrap.configRef</font>
+}
+object Machine << Cluster API CRD >> {
+    <font color="Green">spec.infrastructureRef</font>
+    <font color="Red">spec.bootstrap.configRef</font>
+}
+
+' Bootstrap provider CRD
+object BarBootstrapConfig << Bootstrap provider CRD >> 
+object BarBootstrapConfigTemplate << Bootstrap provider CRD >>
+
+' Infrastructure provider CRD
+object FooMachineTemplate << Infrastructure provider CRD >>
+object FooMachine << Infrastructure provider CRD >>
+
+MachineDeployment "0..*" .right.> "1" Cluster
+MachineDeployment "0..1" ..> "1" FooMachineTemplate
```

Comment:
Should be this 1:1 (Same as MachineSet  1:1 FooMachineTemplate)?

Reply:
I fixed at 73b790f152c08afe6a1e7bfbe107eee99d78e3ec.